### PR TITLE
Helper for Dotenv

### PR DESCRIPTION
**Pull request by petersng**
Pulls out what Dan did in api-server and makes it reusable for multiple projects.

### CODE REVIEW

File: pkg/config/dotenv.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+func PopulateFromDotEnv(envEnvVar string) error {
+	env := os.Getenv(envEnvVar)
+	if "" == env {
+		env = "development"
+	}
+
+	err := godotenv.Load(".env." + env + ".local")
+	if err != nil && !isNoSuchFileErr(err) {
+		log.Errorf("Did not load .env.%v.local: err: %v", env, err)
+		return err
```
Comment (dankins) [inline pkg/config/dotenv.go]: are you sure returning here is the best behavior? The .local is generally used to override and is not required

Comment (dankins) [reply pkg/config/dotenv.go]: (same issue on line 42 and 51) wondering if maybe this should be a debug instead of an error

Comment (petersng) [reply pkg/config/dotenv.go]: So, this particular error return is only in the event that it is *not* a "file not found" error, meaning that the file was loaded, but the data was malformed or some other internal error.  I experienced a situation where I was using a `.env` and while the file was found, the data in the `.env` was malformed. 

Since there was no logging in the old code for the error, just the message that the `.env` file was not loaded, so it was confusing since the file was there.  

So basically in this update, if the file is found but malformed or other, return an error.  If the file is not found, keep going.  If the file is found and is not malformed, keep going and log that the file was loaded.

Does that make sense?

Comment (dankins) [reply pkg/config/dotenv.go]: oh yes makes complete sense, I was just not looking at the code very carefully. :+1:
